blockchains/utxo/utxo_worker.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+const uuidv1 = require('uuid/v1');
+const { logger } = require('../../lib/logger');
+const rp = require('request-promise-native');
```

Comment:
[This NPM](https://www.npmjs.com/package/request-promise-native) has been depricated, should be easy to migrate to `jayson` which we already use in this repo. See 'ETH' blockchain.